.github/workflows/continous-integration-os.patch.yml

```diff
@@ -14,70 +14,42 @@ on:
 jobs:
   test:
     name: Test ${{ matrix.rust }} on ${{ matrix.os }}
-    # The large timeout is to accommodate:
-    # - Windows builds (75 minutes, typically 30-50 minutes)
-    # - parameter downloads (40 minutes, but only when the cache expires)
-    timeout-minutes: 115
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false
       matrix:
         # TODO: Windows was removed for now, see https://github.com/ZcashFoundation/zebra/issues/3801
         os: [ubuntu-latest, macos-latest]
-        rust: [stable]
-
-    steps:
-      - run: 'echo "No build required"'
-
-  test-fake-activation-heights:
-    name: Test ${{ matrix.rust }} zebra-state with fake activation heights on ubuntu-latest
```

Comment:
This matches this suggestion in PR #4637:
https://github.com/ZcashFoundation/zebra/pull/4637/files#r903137662